### ISSUE TITLE
"Error: Not supported" on require('exif')

### ISSUE THREAD

**Issue by Songworks** [MikeKovarik/exifr#24]
Hello there,

I'm getting an error on just import/require of the module. Probably doing something very small wrong but can't figure it out. I boiled a "test setup" down to this:

./src/index.js:
```
const exifr = require('exifr');
// Doing nothing beyond that
```

./package.json:
```
{
  "name": "@testing/test2",
  "private": true,
  "dependencies": {
    "exifr": "^4.3.4"
  }
}
```

`node -v`: v12.16.1
`npm -v`: 6.13.4
OS: Ubuntu 19.10

If I execute `node src/index` (`npm install` beforehand of course) I get the following error:
```
(node:13363) UnhandledPromiseRejectionWarning: Error: Not supported
    at /srv/projects/@testing/test2/node_modules/exifr/dist/full.umd.cjs:1:44218
    at /srv/projects/@testing/test2/node_modules/exifr/dist/full.umd.cjs:1:69
    at Object.<anonymous> (/srv/projects/@testing/test2/node_modules/exifr/dist/full.umd.cjs:1:172)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/srv/projects/@testing/test2/src/index.js:1:15)
(node:13363) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:13363) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

What am I doing wrong?
Also I can require the lite & mini umd.cjs ones without an error. But they don't contain `FSReader` which I probably need.

Anyway appreciate any help and
thanks in advance
Stefan

**Comment by MikeKovarik** [comment]
I just released 4.3.5. That should fix it. If not, please reopen the issue. Thanks

**Comment by Songworks** [comment]
@MikeKovarik Just tried it. Works great. Thank you very much for the fast response & work. Have a great next week!
